src/PrestaShopBundle/Resources/config/services/adapter/pdf/generator.yml

```diff
@@ -32,3 +32,8 @@ services:
 
   prestashop.adapter.pdf.generator.single_invoice:
     class: PrestaShop\PrestaShop\Adapter\PDF\InvoicePdfGenerator
+
+  PrestaShop\PrestaShop\Adapter\PDF\OrderReturnPdfGenerator:
+    arguments:
```

Comment:
```suggestion
    public: false
    arguments:
```

Reply:
Could you elaborate why this should be public?

Reply:
Because all services are meant to be private by default.  Only some services can be public (like controllers for example) but these are exceptions.